Packs/DeprecatedContent/Integrations/Cymon/Cymon.py

```diff
@@ -460,5 +460,5 @@ def http_request(method, url, headers):
     elif command == 'domain':
         cymon_says()
 
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.